### PR TITLE
Create cluster roles for managing ECP resources

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
Let's hold off on merging this until https://github.com/hacbs-contract/enterprise-contract-controller/pull/5 is merged so we can pull that in as well.

File: components/enterprise-contract/kustomization.yaml [hunk 1]
```diff
@@ -2,7 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://github.com/hacbs-contract/enterprise-contract-controller/config/crd?ref=69e8c88a50c471acca37a7eff68e9d129dad08e6
+- https://github.com/hacbs-contract/enterprise-contract-controller/config/crd?ref=1a34e1604a54f439d2515ef92c016521d4208d6b
+# Kustomize does not allow github.com urls that reference a single-file. They must always reference
+# a directory that contains a kustomization.yaml file. The directory /config/rbac does include a
+# kustomization.yaml file but it includes many other RBAC changes that are not desirable here. Use
+# a URL to the "raw" version of the file instead.
```
[the remark this answers is not in the document]
Comment (lcarva) [reply components/enterprise-contract/kustomization.yaml:9]: Agree. I filed [HACBS-625](https://issues.redhat.com/browse/HACBS-625) so we can follow up. There's a lot of other stuff in that repo that I don't quite understand and I want to make sure I give that change my full attention.

**Comment by lcarva** [comment]
/retest

failures seem to be unrelated

**Comment by zregvart** [comment]
Also #434 is of note here. The change in here will require a change in the [Tekton pipeline](https://github.com/hacbs-contract/enterprise-contract-controller/blob/1336dcc40553ef59eda9b681ec96b0b18c06f0dd/.tekton/push.yaml#L19)

**Review by robnester-rh — APPROVED** [review]
LGTM

**Pull request merged**